ios/SmartSyncExplorerReactNative.xcodeproj/project.pbxproj

```diff
@@ -311,7 +311,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "diff \"${PODS_PODFILE_DIR_PATH}/Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [ $? != 0 ] ; then\n    # print error to STDERR\n    echo \"error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\" >&2\n    exit 1\nfi\n# This output is used by Xcode 'outputs' to avoid re-running this script phase.\necho \"SUCCESS\" > \"${SCRIPT_OUTPUT_FILE_0}\"\n";
```

Comment:
cocoapod updated that line - what version are you running?
```shell
$ pod --version
1.2.1
```

Reply:
`1.2.0`. I think it updated that line because I updated the `Xcode` compatibility version of the `Pod` project as well to match ours.